e2e-tests/spring-rest-openapi-v2/src/test/java/org/evomaster/e2etests/spring/examples/authenticatedswaggeraccesstest/UnauthenticatedSwaggerAccessEMTest.java

```diff
@@ -0,0 +1,29 @@
+package org.evomaster.e2etests.spring.examples.authenticatedswaggeraccesstest;
+
+import com.foo.rest.examples.spring.unauthenticatedswaggeraccesscontroller.UnauthenticatedSwaggerAccessController;
+import org.evomaster.e2etests.spring.examples.SpringTestBase;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class UnauthenticatedSwaggerAccessEMTest extends SpringTestBase {
+
+    @BeforeAll
+    public static void initClass() throws Exception {
+
+        UnauthenticatedSwaggerAccessController controller = new UnauthenticatedSwaggerAccessController();
+        SpringTestBase.initClass(controller);
+    }
+
+    /**
+     * Since the swagger endpoint is authenticated, it can be retrieved using authentication object.
+     */
+    @Test
+    public void testRunEM() throws Throwable {
+
+        runTestHandlingFlakyAndCompilation(
+                "UnauthenticatedSwaggerAccessEM",
+                "org.bar.UnauthenticatedSwaggerAccessEM",
+                100,
+                (args) -> initAndRun(args));
+    }
```

Comment:
see previous comments: ie 20 and assertions

Reply:
Hi Andrea,

Thanks for your comment. I have fixed this one.